third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -80,6 +80,11 @@ enum StopAfter {
     FileFormat,
 }
 
+/// Names for 'virtual' processors in the pipeline. This can be used for
+/// filtering via the `config.dump_bytecode_filter` option.
+const INITIAL_BYTECODE_STAGE: &str = "FILE_FORMAT";
```

Comment:
I think you need to explain why the names look like they're reversed.

Reply:
I don't understand this comment.